.github/workflows/internal_vs_external.yml

```
@@ -25,6 +22,7 @@ jobs:
     if: ${{ needs.check-membership.outputs.is_member != 'true' && needs.check-membership.result == 'success' }}
     steps:
       - name: Dismiss Pull Request Reviews
+        if: ${{ github.event.pull_request_target.draft == false }}
```

I find the following easier to read since it maps better to human language. As in: this job runs on the PR
if it's not a draft:
```suggestion
        if: ${{ ! github.event.pull_request_target.draft }}
```